client/my-sites/stats/summary/index.jsx

```diff
@@ -367,6 +368,11 @@ class StatsSummary extends Component {
 			backLink += domain;
 		}
 		const navigationItems = [ { label: backLabel, href: backLink }, { label: title } ];
+		const geoMode = this.props.context.query.geoMode;
```

Comment:
Could we have a Locations module dedicated scope for this query parameter?

```
let pathForCsvFile = path;
if ( module === 'locations' ) {
	const geoMode = this.props.context.query.geoMode;
	const geoModeLabel =
		geoMode && Object.prototype.hasOwnProperty.call( GEO_MODES, geoMode )
			? GEO_MODES[ geoMode ]
			: 'country';
	pathForCsvFile = `${ path }-${ geoModeLabel }`;
}
```

Reply:
Will be even better to put these lines into a little helper 😄